Containerfile

```diff
@@ -69,7 +69,7 @@ LABEL url="https://github.com/rhel-lightspeed/linux-mcp-server"
 LABEL vendor="Red Hat, Inc."
 LABEL version=${VERSION}
 
-ADD LICENSE /licenses/Apache-2.0.txt
+ADD licenses/ /licenses/
```

Comment:
I'm wondering if it would make sense to keep our Apache-2.0 license where it is right now, and only add third-party licenses to the `licenses` folder... Maybe renaming it to `licenses-3rdparty`? 

Not any particular reason, just that if we maintain our license at the root level, it is easier for parsers to identify that this project is associated with Apache 2.0 rather than any of `licenses/*.txt`

Reply:
I was thinking that GitHub supported multiple license files in a top level `licenses/` directory but that doesn't seem to be the case. Let me see what the proper way to do this is. Multiple licenses in a project does complicate things.